external/avro/src/main/scala/org/apache/spark/sql/avro/AvroDataToCatalyst.scala

```diff
@@ -80,22 +80,36 @@ private[avro] case class AvroDataToCatalyst(
   }
 
   @transient private lazy val nullResultRow: Any = dataType match {
-      case st: StructType =>
-        val resultRow = new SpecificInternalRow(st.map(_.dataType))
-        for(i <- 0 until st.length) {
-          resultRow.setNullAt(i)
-        }
-        resultRow
+    case st: StructType =>
```

Comment:
While this whitespace change is OK, it's not necessary, and I'd just avoid it

Reply:
Sure. This indentention changes were done by mistake I'll revert them.